packaging/standalone/standalone-community/src/main/distribution/text/community/conf/neo4j.conf

```diff
@@ -226,6 +226,20 @@ dbms.connector.https.enabled=true
 # clauses that load data from the file system.
 #dbms.security.allow_csv_import_from_file_urls=true
 
+# Value of the HTTP Strict-Transport-Security (HSTS) response header. This header
+# tells browsers that a webpage should only be accessed using HTTPS instead of HTTP.
+# It is attached to every HTTPS response. Setting is not set by default so
+# 'Strict-Transport-Security' header is not sent. Value is expected to contain
+# dirictives like 'max-age', 'includeSubDomains' and 'preload'.
+#dbms.security.http_strict_transport_security=
+
+# Value of the HTTP Public Key Pinning (HPKP) response header. This header tells
+# browsers about the public keys that belong to a webpage. It is attached to every
+# HTTPS response. Setting is not set by default so 'Public-Key-Pins' header is not
+# sent. Value is expected to contain dirictives like 'pin-sha256', 'max-age',
+# 'includeSubDomains' and 'report-uri'.
+#dbms.security.http_public_key_pins=
```

Comment:
We shall emphasize somewhere (maybe docs?) that when using self-signed or internal CA signed certificates, some browsers will not pin public keys.